catroidSourceTest/build.gradle

```diff
@@ -5,7 +5,6 @@ repositories{
 apply plugin: 'java'
 
 dependencies {
-    testCompile files( project(":").buildDir.getPath() + '/intermediates/classes/debug' )
```

Comment:
Some tests need access to the compiled classes instead only on sources. But to find them they need to be declared as dependency.

If those tests have been removed this line is not needed anymore.